http/src/main/resources/reference.conf

```diff
@@ -716,6 +716,19 @@ gke {
     version = "1.28"
     nodepoolLockCacheExpiryTime = 1 hour
     nodepoolLockCacheMaxSize = 200
+
+    autopilot {
+      welder {
```

Comment:
yea, welder and wondershaper is not tunable by users since they user don't interact with them directly...right now, app (SAS and RStudio) resources are passed from API client, and I didn't set defaults for those (will be easy to add if we like later).

Reply:
I didn't provide defaults for these in charts themselves since defining them in leo will be faster for us to iterate these values

Reply:
Then maybe adding a small TODO to move these into the charts once the correct values have been found would be nice

Reply:
I don't know if having defaults in charts provide much value...they'll be overwriten by Leonardo anyway